docs/05-internals/design/principles/futures-with-pekko.md

```diff
@@ -134,7 +137,7 @@ comprehension, you'll already implicitly have a `Future` object.
 
 ## Using `recover` on Futures
 
-By using `recover` on a  `Future`, an apt error message can be thrown if
+By using `recover` on a`Future`, an apt error message can be thrown if
```

Comment:
```suggestion
By using `recover` on a `Future`, an apt error message can be thrown if
```

Reply:
sorry, had automerge enabled... will fix this typo in my other PR, thanks for noticing